### ISSUE TITLE
I played for over an hour and I don't know what I did but here's a record of it.

### ISSUE THREAD

**Issue by Satoshi0x** [dacoinminster/BabyMerchant#1]
I know you wrote the original Mastercoin whitepaper. So I tested your game. It confuses me. but I hope my test helps your goals with this project.

Thanks for your work

18,305 ƀ - lv 1
[Baby Merchant.pdf](https://github.com/dacoinminster/BabyMerchant/files/7989305/Baby.Merchant.pdf)

`18A9NWXTgxjJUz82UAqQsb1bjk2XrUUmMj`

[Daniel R. Treccia](https://bitcointalk.org/index.php?action=profile;u=2226756) (satoshi0x)

**Comment by dacoinminster** [comment]
lol It's a game I made to amuse my children. There's definitely a lot of things that could be improved. Thanks for your feedback. :)

**Comment by Satoshi0x** [comment]
> ƀ

In the spirit of my crypto and meta obsession when it comes to globalized communications - and a love for simple text or even unicode and their fun little "emojis" I left you my Bitcoin legacy address (Base 58 key) for a laugh. Because the points I was able to accrue in a long time of clicking and critical thinking how to do everything the best "buy low, sell high!" (albeit in boogers) were denominated in a lowercase Latin character I had seen before, and I used Microsoft Bing to search in Edge in the spirit of GitHub being the site of this here game (hat tip to the parent who bought this information trove of talented people like yourself, I have been a fan of you for a decade now!):

![image](https://user-images.githubusercontent.com/48847567/152261593-b0f4b442-f8ea-4755-a573-52f50b949bc0.png)

![image](https://user-images.githubusercontent.com/48847567/152261763-d8908e41-ec29-4503-b4ed-9e9c5f73b1c8.png)

I have no idea if I'm crazy or if we're just that close to this whole thing making sense but what I do know is anything that seems silly by an innovator I highly revere is worth my time. I came from the Devcoin Project where they let me make outreach to the community with official news and updates. It's an honor. But as for the lowercase b with its strike. Well... I won't ask for the answers directly but I am giving you feedback that this type of work is important all over the globe. I was just [reading into Kanji and how the w3 working group](https://lists.w3.org/Archives/Public/www-svg/2010Jun/0143.html) was working on its external characters from the official unicode set back then - the set was yet to be included in Unicode but was in many Japanese books "Gaiji"... Since communication and code are both beautiful and core to correct and secure flow of information over the entire connected globe, I found it relevant and also reminded me that you  have a text game, with simple shapes, two baby emojis (as we call them now) prob from some unicode release that now, thanks to smart phones and standards, are globally recognized forms of communication.

I brought up Gaiji and text art standards etc because recently I contributed to making what our lead dev at Devcoin will be using for the splash screen logo on our upcoming Devcoin Core 22..x testnet. I realized the SVG file was made by text itself under the hood and it was made in Adobe Illustration 15.0.0. SVG Fonts are also a thing in text. That's where I read about the plans in 2010 to fix this "Gaiji" external character issue for Japanese books not translating over to the Latin Script using countries bc of a little thing called Unicode standards or "emojis" - since 2010 part of everyday smartphone life!

I have been enjoying my foray into projects like Monacoin, Koto, etc, and learning about the meme equivalents to North American ones from the early days of the web like Pepe's (because of their explosion in value on Bitcoin blockchain/network) and the Japanese textboards like "2channel" that used a JP version of what we call ASCII art here. That being said, since you made a way to extend Bitcoin by writing a document that forever changed how Bitcoin can be used and preceded Counterparty where massive success is being had with collectables - and even your doc leading to the "ICO" type launch of Property #31 on Mastercoin >> Omni Protocol (known as TetherUS) - I think anything you make simple or complex has at the very least a reason to provide feedback.

I don't do this often. I gotta be honest I hate gaming. But I loved the after effects of reviewing and learning about what I just did.

So thank you, and I hope we hear more from you in the near future.

If you ever can, we'd love for you to drop by our developers official community chat on telegram for Devcoin (devcoin.org since 2011) - https://t.me/Devcoins

-Dan T.
Devcoin "A-Team"
Community Manager

**Comment by Satoshi0x** [comment]
![333](https://user-images.githubusercontent.com/48847567/152263321-6b522f4a-3b57-407a-92e8-899c4b6a8cce.png)

This is my first actual contribution to our project official codebase so I was happy :) thought I'd humble brag

**Comment by dacoinminster** [comment]
Yes it amused me to use ƀ for my booger currency unit because of stuff like that. Bit of an inside joke, and I'm pleased you noticed :)

**Comment by overlyGreenBanana** [comment]
I have found you at last
